### ISSUE TITLE
Build MOI splash page for MWC

### ISSUE THREAD

**Issue by edrushka** [mozilla/webmaker.org#1168]
Depends on #1167 (design) and #1171 (Cassie's signoff on Design)

**Comment by mmmavis** [comment]
@edrushka have we decided the path for this MOI splash page?

**Comment by sabrinang** [comment]
__GDoc Copy:__ https://docs.google.com/a/mozillafoundation.org/document/d/1d2ET2yBrM0C_ObUPyhqkjTWAKgbzjd7UGP8tMhslKsc/edit

__Design files for reference:__ https://www.dropbox.com/sh/5i7ijrze13z5y5f/AAAmnq3pGOfTqZRO_24vSEBHa?dl=0
_1 - Projects > Mobile Opportunity Initiative > Design Files_

__Assets:__ https://www.dropbox.com/sh/26xiidgm6w44tll/AAB2u2-BroA5Ft-AokrGsRNVa?dl=0
_1 - Projects > Mobile Opportunity Initiative > Assets_
*NOTE: mobile hand & globe illustration subject to change - will update and export into this folder

__Icons:__ http://fontawesome.io/
__Flags:__ http://lipis.github.io/flag-icon-css/
__Colours:__ http://mozilla.github.io/makerstrap/demo/#/
__Partner Icons:__ to be confirmed

__Illustration/Spacing adjustments feedback__ (not a blocker - will be updating this)
_Desktop:_ https://redpen.io/sveed4aec568a4488a
_Mobile:_ https://redpen.io/wjc1c9851c3fd2c80a

**Comment by mmmavis** [comment]
@sabrinang yay thanks! :smile:

**Comment by mmmavis** [comment]
Team has decided to use webmaker.org/localweb as the URL.

**Comment by mmmavis** [comment]
:dancers:  Check out staging here: https://moi-splash.herokuapp.com/localweb (page WIP still)

**Comment by mmmavis** [comment]
@edrushka @adamlofting @thisandagain 
Not sure if this has been discussed already - do we need to plug in a **new** GA tracking code on this page?

**Comment by adamlofting** [comment]
>  Not sure if this has been discussed already - do we need to plug in a new GA tracking code on this page?

@mmmavis thanks for checking :+1: 

If the final URL where this is hosted is ` webmaker.org/localweb` (or hosted anywhere on webmaker.org or a subdomain), we will use the same GA tracking code as used on the rest of webmaker.org. 

Q1: Looking at the source of that demo URL, this looks like it's been built into the webmaker.org repo, is that correct?

If yes, the GA tracking code will appear when the environment variables are set: https://github.com/mozilla/webmaker.org/blob/master/views/layout.html#L188 

After that, we will want to implement a couple of custom tracking features for the links that point to external URLs (app download, and PDFs).

**Comment by mmmavis** [comment]
@adamlofting (yeah another markdown syntax learned! ">")
> Q1: Looking at the source of that demo URL, this looks like it's been built into the webmaker.org repo, is that correct?

I'm reusing w/e the codebase offers but I'll have the Webmaker devs to do reviews for me to make sure.  Thanks!

**Comment by adamlofting** [comment]
Thanks @mmmavis.

For the next stage we need to add some custom event tracking to the links on the page that take the user to 'non-webpages'. I've written a spec for this in a separate ticket:

- [ ] Track links to non-webpage URLs in MOI Splash Page https://github.com/mozilla/webmaker.org/issues/1254

**Comment by edrushka** [comment]
@mmmavis can we close this?

**Comment by mmmavis** [comment]
yup!
